examples/linear_regression/README.md

```diff
@@ -0,0 +1,25 @@
+# Dot Product Tutorial
+
+[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/NillionNetwork/nada-numpy/blob/main/examples/dot_product/linear_regression.ipynb)
```

Comment:
@jcabrero this link should end in `linear_regression/linear_regression.ipynb` instead of `dot_product/linear_regression.ipynb`